src/menu/menu.scss

```diff
@@ -162,6 +162,10 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
   }
 }
 
+@mixin iui-menu-content {
+  padding: $iui-baseline ($iui-component-padding-horizontal + 1px);
```

Comment:
Why +1?

Reply:
It's the same horizontal padding used within `@mixin iui-menu-item` on [line 44](https://github.com/iTwin/iTwinUI/blob/d2d3e42568500dec2a60a174909867ebacd96924/src/menu/menu.scss#L44).  I think the +1 was used to accommodate for the border?